pkg/sql/backfill/backfill.go

```diff
@@ -214,7 +229,16 @@ func (cb *ColumnBackfiller) InitForDistributedUse(
 	// entire backfill process.
 	flowCtx.TypeResolverFactory.Descriptors.ReleaseAll(ctx)
 
-	return cb.init(evalCtx, defaultExprs, computedExprs, desc)
+	return cb.init(evalCtx, defaultExprs, computedExprs, desc, mon)
+}
+
+// Close frees the resources used by the ColumnBackfiller.
+func (cb *ColumnBackfiller) Close(ctx context.Context) {
+	cb.fetcher.Close(ctx)
+	if cb.mon != nil {
```

Comment:
If we're keeping this for a safeguard, should we also be checking that `boundAccount` is set?